pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
```

Comment:
Mind just adding a comment above this variable explaining its role, also just thinking maybe `returnHref` is a better name just so its clear it is a url. (The naming is minor if you don't agree feel free to leave it).